packages/x-components/src/x-modules/experience-controls/store/actions/fetch-and-save-controls.action.ts

```diff
@@ -1,11 +1,12 @@
+import { ExperienceControlsRequest, ExperienceControlsResponse } from '@empathyco/x-types';
+import { Dictionary } from '@empathyco/x-utils';
 import { createFetchAndSaveActions } from '../../../../store/utils/fetch-and-save-action.utils';
 import { ExperienceControlsActionContext } from '../types';
 
 const { fetchAndSave, cancelPrevious } = createFetchAndSaveActions<
   ExperienceControlsActionContext,
-  //TODO: change types when the adapter is updated
-  any,
-  any
+  ExperienceControlsRequest,
+  ExperienceControlsResponse | Dictionary<any>
```

Comment:
This has to be just `ExperienceControlsResponse`, you have to change the type in the `types.ts` file of the module and adapt the return of the `fetchExperienceControlsResponse` action.

```suggestion
  ExperienceControlsResponse
```